### PR TITLE
Add EthStats docs

### PR DESCRIPTION
**Pull request by baxy**
Add documentation on how to install the EthStats tools:
- Lite Block Explorer
- Lite Network Health Monitor

### CODE REVIEW

File: docs/EthStats/Lite-Network-Health-Monitor.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Github repositories
+* [ethstats-cli](https://github.com/Alethio/ethstats-cli) - client app that extracts data from a node and sends it to the server 
+* [ethstats-network-server](https://github.com/Alethio/ethstats-network-server) - service that consumes the data received from the nodes through the client app 
+* [ethstats-network-dashboard](https://github.com/Alethio/ethstats-network-dashboard) - front end dashboard  
+
+## Installation
+
+The installation process is done through multiple steps. First the server needs to be installed with its dependencies and then for every node in the network the client app `ethstats-cli` is needed.
+
+#### Server
```
Comment (NicolasMassart) [inline docs/EthStats/Lite-Network-Health-Monitor.md]: I don't see where you make the link between the server and the Pantheon node. Can you clarify this point ?

File: docs/EthStats/Lite-Network-Health-Monitor.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* [Redis](https://redis.io/) - if persistence is needed
+
+More details about installing and running are available in the github [repository](https://github.com/Alethio/ethstats-network-server).
+#### EthStats-CLI
+
+After the server is up and running the next step is to install `ethstats-cli` for the nodes in the network. This can be done through: 
+* [NPM](https://github.com/Alethio/ethstats-cli#install) - the package needs to be installed globally and also can be used as a [daemon](https://github.com/Alethio/ethstats-cli#daemon)
+* [Docker](https://github.com/Alethio/ethstats-cli#docker) - images available on [docker hub](https://hub.docker.com/r/alethio/ethstats-cli)
+
+More details about installing and running are available in the github [repository](https://github.com/Alethio/ethstats-cli).
```
Comment (MadelineMurray) [inline docs/EthStats/Lite-Network-Health-Monitor.md]: It'd be really nice to see an end to end example (either using Docker or installing) of setting these things up with Pantheon.  Do you think that's something could be included?

Comment (baxy) [reply docs/EthStats/Lite-Network-Health-Monitor.md]: Sure thing, added examples.

File: docs/EthStats/Lite-Network-Health-Monitor.md (hunk 1 above)
Comment (baxy) [reply docs/EthStats/Lite-Network-Health-Monitor.md]: To bring more clarity, I added installation examples that includes the need of adding the `--server-url` flag and also `--client-url` if the Pantheon node is not running on the same host as `ethstats-cli`.

File: docs/EthStats/Lite-Network-Health-Monitor.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+historical statistics about the network nodes.
+
+The lite version supports in-memory persistence or using Redis to persist a fixed number of blocks (by default,
+3000). 
+
+View the Network Health Monitor for the [Ethereum MainNet](https://net.ethstats.io)
+
+!!! note 
+     The EthStats Lite Block Explorer is an [Alethio product](https://aleth.io/).
+
```
Comment (NicolasMassart) [inline docs/EthStats/Lite-Network-Health-Monitor.md]: Also perhaps note the list of ports that will be used in this tutorial as we use static local port 80 and 3000 and there's many chances of conflicts with existing services.

File: docs/EthStats/Lite-Network-Health-Monitor.md [hunk 4]
```diff
@@ -3,72 +3,117 @@ description: Lite Network Health Monitor
...
+2. Change into the `/ethstats-network-server/docker/lite-mode/memory-persistence` directory:
+   
+    ```bash
+    cd ethstats-network-server/docker/lite-mode/memory-persistence
+    ```
+
+3. Use the provided `docker-compose` file to start the server: 
+
+    ```bash
+    docker-compose up
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/EthStats/Lite-Network-Health-Monitor.md]: Fixed

File: docs/EthStats/Lite-Network-Health-Monitor.md [hunk 5]
```diff
@@ -3,72 +3,117 @@ description: Lite Network Health Monitor
...
+!!! tip
+    A `docker-compose` file is also provided in the `ethstats-network-server/docker/lite-mode/redis-persistence`
+    directory to run the server using Redis to persist a fixed number of blocks (default is 3000).
+
+### 2. Pantheon 
+
+Start Pantheon in development mode with Websockets enabled:
+
+```bash
+docker run -p 8546:8546 --mount type=bind,source=/<pantheondata-path>,target=/var/lib/pantheon pegasyseng/pantheon:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-cors-origins="all" --rpc-ws-enabled --network=dev
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/EthStats/Lite-Network-Health-Monitor.md]: Fixed

File: docs/EthStats/Lite-Network-Health-Monitor.md [hunk 6]
```diff
@@ -3,72 +3,117 @@ description: Lite Network Health Monitor
...
+```
+
+Where `<pantheondata-path>` is the volume to which the node data is saved. 
+
+### 3. Client 
+
+Start the client for the Pantheon node:  
+
+```bash
+docker run -it --restart always --net host -v /<configurationfiles-path>/opt/ethstats-cli/:/root/.config/configstore/ alethio/ethstats-cli --register --account-email <email> --node-name <node_name> --server-url http://localhost:3000 --client-url ws://127.0.0.1:8546
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/EthStats/Lite-Network-Health-Monitor.md]: Fixed

File: docs/EthStats/Lite-Network-Health-Monitor.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/EthStats/Lite-Network-Health-Monitor.md]: Added

File: docs/EthStats/Lite-Network-Health-Monitor.md [hunk 7]
```diff
@@ -3,72 +3,117 @@ description: Lite Network Health Monitor
...
+Registering the node is only required the first time the client is started for the node.
+* `--client-url` specifies the WebSockets URL for the Pantheon node.    
+
+### 4. Dashboard 
+
+To display the Network Health Monitor dashboard, open [`localhost`](http://localhost) in your browser. 
+
+### Stopping and Cleaning Up Resources
+
+When you've finished running the Network Health Monitor, stop and delete the containers.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/EthStats/Lite-Network-Health-Monitor.md]: Fixed

**Pull request merged**